worlds/mmx3/Rules.py

```diff
@@ -0,0 +1,451 @@
+from worlds.generic.Rules import add_rule, set_rule
+from BaseClasses import CollectionState
+
+from . import MMX3World
```

Comment:
I'm kind of surprised this isn't giving you a recursive import error

Reply:
I'm surprised as well now that I look at it, I think this has been around for a while in the world.

Reply:
Well, as long as it works I guess lol